superset/config.py

```diff
@@ -1129,8 +1129,10 @@ def CSV_TO_HIVE_UPLOAD_DIRECTORY_FUNC(  # pylint: disable=invalid-name
 TRACKING_URL_TRANSFORMER = lambda url: url
 
 
-# Interval between consecutive polls when using Hive Engine
-HIVE_POLL_INTERVAL = int(timedelta(seconds=5).total_seconds())
+# customize the polling time of each engine. The default time is 5 seconds
+DB_POLL_INTERVAL_SECONDS = {
+    "hive": int(timedelta(seconds=5).total_seconds()),
+}
```

Comment:
We can't really define defaults in `config.py`. Imagine if I want to override the poll interval for "impala" and add the following to `superset_config.py`:

```
DB_POLL_INTERVAL_SECONDS = {"impala": 1}
```

In this case the default for "hive" in `config.py` would be lost. This is why I recommended not defining the defaults in `config.py`, and rather placing them in their respective db engine specs.

Reply:
OK, thank you for your suggestion. I can let the user set it without setting the default value. I want to change it to this?
DB_POLL_INTERVAL_SECONDS = {}

Reply:
I would add the type as that's the convention in `config.py` and mypy can't infer the type from an empty dict:
```python
DB_POLL_INTERVAL_SECONDS: Dict[str, int] = {}
```

Reply:
I changed it to this way. It's true that mypy failed